Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonHelpCommand.cs

```diff
@@ -18,28 +18,29 @@ public class KryptonHelpCommand : KryptonCommand
     {
         #region Instance Fields
 
-        private ButtonSpecAny _helpButtonSpec;
+        private ButtonSpecAny? _helpButtonSpec;
 
         #endregion
 
         #region Public
 
-        public ButtonSpecAny HelpButton
+        /// <summary>Gets or sets the help button.</summary>
+        /// <value>The help button.</value>
+        [DefaultValue(null), Description(@"Access to the help button spec.")]
+        public ButtonSpecAny? HelpButton
         {
-            get => _helpButtonSpec;
+            get => _helpButtonSpec ?? new();
             set { _helpButtonSpec = value; UpdateImage(KryptonManager.InternalGlobalPaletteMode); }
         }
 
         #endregion
 
         #region Identity
 
+        /// <summary>Initializes a new instance of the <see cref="KryptonHelpCommand" /> class.</summary>
         public KryptonHelpCommand()
         {
-            switch (KryptonManager.InternalGlobalPaletteMode)
-            {
-
-            }
+            _helpButtonSpec = null;
```

Comment:
This is not needed as it already will be null when created.

Reply:
Addressed feedback